app/services/helpers/custom-round.js

```diff
@@ -0,0 +1,15 @@
+module.exports.customRound = function (number) {
```

Comment:
Since this file only has one function you can use `module.exports = function (number) {` and then when you `require` it, the import acts as the function, ie. 

`const customRound = require('./custom-round')`
`capacityPercentageRow.values.push(customRound(capacity['capacity_percentage']))`